docs/content/en/getting_started/architecture.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Architecture"
+description: "DefectDojo consists of several components that work together closely."
+draft: false
+weight: 1
+---
+
+![Architecture](../../images/DD-Architecture.svg)
+
+## NGINX
+
+The webserver [NGINX](https://nginx.org/en/) delivers all static content, e.g.
+images, JavaScript files or CSS files.
+
+## uWSGI
+
+[uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/) is the application server
+that runs the DefectDojo application, written in Python/Django, to serve all
+dynamic content.
+
+## RabbitMQ
+
+The application server sends tasks to a [RabbitMQ](https://www.rabbitmq.com/)
+message queue for asynchronous execution. 
```

Comment:
@StefanFl the architecture page is not accurate. DefectDojo is **not strongly linked** to RabbitMQ.
DefectDojo rely on Celery. Celery can leverage RabbitMQ for message passing but this can be replaced by Redis or another system.
I would prefer to have something more high level in this page. If you really want to talk about implementation, you can redirect users to https://docs.celeryproject.org/en/stable/getting-started/brokers/index.html#broker-overview

Reply:
I agree with @damiencarol, we should have the term `broker` maybe instead, and list rabbitMQ as being possibly the default for us.

Reply:
Yes, having a Message Broker in the diagram, with a reference to RabbitMQ and Redis in the text is much better.

And can do some double-edge arrows as well.